docs/personalization/recommendation_models.md

```diff
@@ -34,6 +34,11 @@ Models from this category return the most popular content items/products, based
 usage history (recent events are more important) and category-based filtering (bestsellers 
 in the selected category and/or subcategories).
 
+#### Predictive
+
+Predicts user's purchase behavior and trends, and recommends items based on this behavior.
```

Comment:
users'
In case of predictive model, all purchases or clicks of particular item are taken into account